Docs/source/usage/parameters.rst

```diff
@@ -2098,7 +2098,11 @@ Details about the collision models can be found in the :ref:`theory section <mul
       This implements the generation of electron-positron pairs based on the analytical cross-section, e.g.
       equation (1) in Gould. The angular distribution of the emitted pairs is isotropic for now
       (instead of following the correct distribution, see e.g. `Ribeyre et al. (Plasma Phys. Control. Fusion 60 104001, 2018) <https://doi.org/10.1088/1361-6587/aad6da>`__).
-      The implementation follows the same numerical algorithm as that of fusion reactions (see. `Higginson et al. (JCP 388, 439-453, 2019) <https://doi.org/10.1016/j.jcp.2019.03.020>`__).
+      The implementation follows the same numerical algorithm as that of fusion reactions (see. :cite:t:`param-HigginsonJCP2019`).
+    - ``linear_compton`` for linear Compton scattering between a lepton (electron or positron, for now) and a photon, based on the Klein-Nishina cross-section
```

Comment:
Most people know "linear Compton" as simply "Compton scattering" (and "non-linear Compton" as "Quantum synchrotron" or "multi-photon Compton") ; maybe it would be good to clarify in some form here, so that people would know what we are talking about.